R/rdeephaven/tests/testthat/test_client_wrapper.R

```diff
@@ -0,0 +1,204 @@
+library(testthat)
+library(rdeephaven)
+
+setup <- function() {
+
+    df1 <- data.frame(string_col = c("I", "am", "a", "string", "column"),
+                int_col    = c(0, 1, 2, 3, 4),
+                dbl_col    = c(1.65, 3.1234, 100000.5, 543.234567, 0.00))
+
+    df2 <- data.frame(col1 = rep(3.14, 100),
+                        col2 = rep("hello!", 100),
+                        col3 = rnorm(100))
+
+    df3 <- data.frame(matrix(rnorm(10 * 1000), nrow = 10))
+
+    df4 <- data.frame(time_col = seq.POSIXt(as.POSIXct(Sys.Date()), as.POSIXct(Sys.Date()+30), by = "1 sec")[250000],
+                    bool_col = sample(c(TRUE, FALSE), 250000, TRUE), 
+                    int_col = sample(0:10000, 250000, TRUE))
+
+    return(list("df1" = df1, "df2" = df2, "df3" = df3, "df4" = df4))
+}
+
+##### TESTING GOOD INPUTS #####
+
+test_that("client connection works in the simple case of anonymous authentication", {
+
+    # assumes correctness of client options
+    client_options <- ClientOptions$new()
```

Comment:
The new default PSK authentication will probably break this in CI, unless @niloc132 has the test workers starting up unauthenticated.

Reply:
Probably yes. But without any of the CI stuff set up, I don't think it makes sense to try to write tests for the PSK case yet. Should we worry about this when we get to CI?

Reply:
Should have a CI ticket linked here.

Reply:
Actually, don't worry about a ticket.  This will appear when the tests are hooked into CI.